packages/postgres/src/test-utils.ts

```diff
@@ -0,0 +1,153 @@
+import pg from 'pg';
+
+import * as defaultPool from './default-pool';
+
+const POSTGRES_USER = 'postgres';
+const POSTGRES_HOST = 'localhost';
+const POSTGRES_DATABASE = 'postgres';
+
+export interface PostgresTestUtilsOptions {
+  user?: string;
+  host?: string;
+  defaultDatabase?: string;
+  database: string;
+  prepareAfterReset?: (client: pg.Client) => Promise<void>;
+}
+
+interface CreateDatabaseOptions {
+  dropExistingDatabase?: boolean;
+  database?: string;
+  templateDatabase?: string;
+  configurePool?: boolean;
+  prepare?: (client: pg.Client) => Promise<void>;
+}
+
+interface DropDatabaseOptions {
+  database?: string;
+  force?: boolean;
+  closePool?: boolean;
+}
+
+async function createDatabase(
+  options: PostgresTestUtilsOptions,
+  createOptions: CreateDatabaseOptions = {}
```

Comment:
To help me understand the patterns we are using here, could you have kept all the defaults together (and simplified the later code) with something like the following?
```suggestion
  {dropExistingDatabase = true, configurePool = true, ...more options... }: CreateDatabaseOptions
```

Reply:
Yep, that would be equivalent. I'll make that change!